### PR TITLE
"tabs" permission not needed

### PR DESCRIPTION
**Pull request by nairb774**
The current implementation doesn't seem to use the "tabs" permission.

Per the documentation at https://developer.chrome.com/extensions/tabs none of the restricted APIs seem to be used.

### CODE REVIEW

**Comment by benblack86** [comment]
Isn't it needed to create tabs? https://github.com/benblack86/linkclump/blob/master/src/background.js#L28

**Comment by nairb774** [comment]
I thought so too, but looking at the documentation, they only seem to call out a few places that actually need the permission. For example, you can see this under the `url`, `title`, and `favIconUrl` properties of the `Tab` object: https://developer.chrome.com/extensions/tabs#type-Tab

Cloning the extension locally, removing the permission and loading the unpacked version into my browser seems to function in terms of being able to open tabs. I can't confirm all functionality works [1], but I was at least able to make that one basic action work. I also compared all uses of the `chrome.tabs` APIs and objects against the documentation to make sure that there wasn't anything lurking.

In short, I think this is safe, but please don't assume I was able to test or understand the code fully. I gave it a good 30 minute try. :)

[1] Not sure I know or use all of the functions available.

**Comment by benblack86** [comment]
🤞
